### DISCUSSION TITLE
Wind Charges Setting

### DISCUSSION THREAD

**Discussion by y-dobon** [GriefPrevention/GriefPrevention#2397] [Ideas]
**As** a server admin
**I would like** a setting in the grief prevention plugin that makes you able to disable the usage of Wind Charges by untrusted players in a claim (like the enderpearls)
**Because** I want my players to stop ruining others homes and mess up the (trap)doors with these new Wind Charge items.

Optional questions:

**Is your feature request related to a problem? Please describe.**
The problem is that now people can freely use the Wind Charges and blow open doors and trapdoors, which gets very annoying when trying to keep people out of your house, or to endlessly fix your trapdoor decorations.

**Describe alternatives you've considered**
For now I have made sure our spawners don't spawn Breezes, so people can't obtain these Wind Charges, but that also means they can't collect the Heavy Core to craft a Mace, which for now I have conveniently put in our vote crates.. which ain't ideal but it works.

**Comment by Jikoo** [comment]
https://github.com/GriefPrevention/GriefPrevention/issues/2357#issuecomment-2295323514
> v17 handles this as of 2330e5441f7a30c33bc3c1dc12f2c65078878e73.
> 
> If anyone is interested in backporting this to legacy, the commit of just wind charge handling was 2ff8d744bd1f9fc0530bd232b2d20454bf223148. Requires updating build processes to use Java 21 and Spigot 1.21.

**Reply by y-dobon** [reply]
Thank you for the reply!

Does this mean I should wait till version 17 comes out? I looked at the 2330e54 and 2ff8d74 but I'm still learning how to maintain all the plugins and am not sure what to do this those :x

**Reply by Jikoo** [reply]
Personally, I'm of the opinion that v17 is fine to use while it's in development so long as you test functionality before putting it on a live server. Commits generally have pretty descriptive messages, so you can read the history somewhat like a changelog and see if you're okay with the features cut (the WG integration, siege, and nature restoration are gone, /givepet is an addon, etc.). Commits since v16 are visible here: https://github.com/GriefPrevention/GriefPrevention/compare/da38fbdf941353e99fb8900072b313a1e6bc676c...master
